namer/consul/src/main/scala/io/buoyant/namer/consul/ConsulInitializer.scala

```diff
@@ -47,7 +51,8 @@ case class ConsulConfig(
   setHost: Option[Boolean] = None,
   consistencyMode: Option[ConsistencyMode] = None,
   failFast: Option[Boolean] = None,
-  preferServiceAddress: Option[Boolean] = None
+  preferServiceAddress: Option[Boolean] = None,
+  weights: Option[Set[TagWeight]] = None
```

Comment:
This should be an `Option[Seq[TagWeight]]` since this is coming from a json list and duplicates will be eliminated anyway when this is transformed into a map.